pkg/minikube/constants/constants.go

```diff
@@ -39,4 +39,14 @@ func MakeMiniPath(fileName string) string {
 	return filepath.Join(Minipath, fileName)
 }
 
+var LogFlags = [...]string{
```

Comment:
What dp you think about making this a struct instead. Something like:

type GlogFlags struct {
  LogToStderr bool
  AlsoLogToStderr bool
...
}


Then you can define a ToFlags method that turns it into a string of flag values.

Then the GetStartCommand can take that config struct, call .ToFlags() and use a simple format string.